lace/emulator/gp_emulator.py

```diff
@@ -113,9 +115,9 @@ def _training_points_polyfit(self,archive):
         coefficients '''
 
         self._fit_p1d_in_archive(4,self.kmax_Mpc)
```

Comment:
Didn't we have now an option to change the polynomial order? How is this still hardcoded then?

Reply:
I don't have it hardcoded in the local version I pushed... 

```python
    def _training_points_polyfit(self,archive):
        ''' Method to get the Y training points in the form of polyfit 
        coefficients '''

        self._fit_p1d_in_archive(self.ndeg,self.kmax_Mpc)
        coeffs=np.empty([len(getattr(self.archive, self.key_data)),self.ndeg+1])
        for aa in range(len(getattr(self.archive, self.key_data))):
            coeffs[aa]=getattr(self.archive, self.key_data)[aa]['fit_p1d'] ## Collect P1D data for all k bins

        return coeffs
``` 

It is here you are looking, right?

Reply:
No, this is not the version I'm reviewing... I'm reviewing the PR by Jonas on the new postprocessing, not the PR with the NN emulator.

Reply:
I guess that the PR by Jonas does not address the changes in polynomial order, what is fine. This will be added in the NN PR, I guess.

Reply:
ah ok, then these two versions are not merged yet...